.github/workflows/ci.yml

```diff
@@ -165,8 +176,16 @@ jobs:
     secrets: inherit
   test-ui:
     name: Test UI
-    # The test-ui Job is only run for pushes to main and pull requests, ui/*, backport/ui/*, release/*, and merge*
-    if: github.ref_name == 'main' || startsWith(github.ref_name, 'ui/') || startsWith(github.ref_name, 'backport/ui/') || startsWith(github.ref_name, 'release/') || startsWith(github.ref_name, 'merge') || github.head_ref == 'main' || startsWith(github.head_ref, 'ui/') || startsWith(github.head_ref, 'backport/ui/') || startsWith(github.head_ref, 'release/') || startsWith(github.head_ref, 'merge')
+    # The test-ui job is only run on:
+    # - pushes to main and branches starting with "release/"
+    # - PRs where the branch starts with "ui/", "backport/ui/", "merge", or when base branch starts with "release/"
+    if: |
+      github.ref_name == 'main' ||
+      startsWith(github.ref_name, 'release/') ||
+      startsWith(github.base_ref, 'release/') ||
```

Comment:
This is redundant. It means that everyone back-porting non-UI changes would now run the test-ui job, but they didn't need to run it for their original PR against the _main_ branch. In addition, I think the whole purpose of branches starting with `backport/ui/` are meant to capture those cases where a change that requires UI testing is being proposed in a PR against a release branch.